system_modules/naprender/src/shader.cpp

```diff
@@ -574,11 +574,11 @@ static bool addShaderVariablesRecursive(nap::ShaderVariableStructDeclaration& pa
 
 	for (int index = 0; index < type.member_types.size(); ++index)
 	{
-		spirv_cross::SPIRType member_type = compiler.get_type(type.member_types[index]);
+		auto member_type = compiler.get_type(type.member_types[index]);
```

Comment:
I understand that sometimes it is better or more convenient to use `auto`, but in this case the type is rather useful and short enough to type out. It's therefore not necessary to rename it to `auto`.